### PR TITLE
[BH-1474] Randomize quotes

### PR DESCRIPTION
**Pull request by paweljonskim**
Randomize quotes

**Description**

**Your checklist for this pull request**

Make sure that this PR:
- [X] Complies with our guidelines for contributions
- [X] Has unit tests if possible.
- [X] Has documentation updated

### CODE REVIEW

File: products/BellHybrid/apps/application-bell-main/presenters/HomeScreenPresenter.cpp [hunk 1]
```diff
@@ -122,7 +123,7 @@ namespace app::home_screen
     {
         stateController->resetStateMachine();
         getView()->setTimeFormat(timeModel->getTimeFormat());
-        getView()->setTime(timeModel->getCurrentTime());
+        getView()->setTime(23 * 60 * 60 + 12 * 60);
```
Comment (mjan-dev) [inline products/BellHybrid/apps/application-bell-main/presenters/HomeScreenPresenter.cpp:126]: Consider adding constexpr for explicit information.

File: products/BellHybrid/apps/application-bell-main/presenters/HomeScreenPresenter.cpp [hunk 2]
```diff
@@ -245,4 +246,16 @@ namespace app::home_screen
         app->switchWindow(gui::BellBatteryStatusWindow::name,
                           std::make_unique<gui::BellBatteryStatusWindow::Data>(getBatteryLvl(), isBatteryCharging()));
     }
+
+    UTF8 HomeScreenPresenter::getGreeting()
+    {
+        const auto greetingCollection = utils::translate_array("app_bell_greeting_msg");
+        if (greetingCollection.empty()) {
+            LOG_WARN("app_bell_greeting_msg array does not exist, using default string");
+            return "app_bell_greeting_msg";
```
Comment (mjan-dev) [inline products/BellHybrid/apps/application-bell-main/presenters/HomeScreenPresenter.cpp:255]: Shouldn't it return some default greeting, not "app_bell_greeting_msg"?

File: products/BellHybrid/apps/application-bell-main/presenters/HomeScreenPresenter.cpp (hunk 1 above)
Comment (paweljonskim) [reply products/BellHybrid/apps/application-bell-main/presenters/HomeScreenPresenter.cpp:126]: oh well, that is not a part of this PR, I commited old local change. Removing!

File: products/BellHybrid/apps/application-bell-main/presenters/HomeScreenPresenter.cpp (hunk 2 above)
Comment (paweljonskim) [reply products/BellHybrid/apps/application-bell-main/presenters/HomeScreenPresenter.cpp:255]: This is the situation when our greetingCollection is empty - that means we have no greetings in current language. Displaying the string identifier is coherent with the rest of the system - if any string is missing in current translation - then we display the string identifier.

Do you suggest hardcoding some greeting in the code? I believe that would mean an English default greeting for any current language

**Review by mjan-dev — APPROVED** [review]

**Review by swichu91 — APPROVED** [review]

**Pull request merged**